docs/fragments/lib/android.md

```diff
@@ -1,15 +1,17 @@
-## Amplify Android (Preview)
+## Amplify Android
 
-This guide shows how to build an app using our Amplify Libraries for Android (Preview) and the Amplify CLI toolchain. To use the existing AWS Mobile SDK for Android, refer to the [AWS Mobile SDK for Android guide](~/sdk/sdk.md).
+This guide shows how to build an app using our Amplify Libraries for Android and the Amplify CLI toolchain.
 
 <docs-internal-link-button href="~/lib/project-setup/prereq.md">
   <span slot="text">Get Started 🚀</span>
 </docs-internal-link-button>
 
+## AWS Mobile SDK for Android
+We highly recommend that you adopt Amplify, but if you are still using our low level SDK for Android, you can find the documentation here: [AWS Mobile SDK for Android](~/sdk/sdk.md).
```

Comment:
This shouldn't block this pull, because we can bikeshed this forever, but I'd like to get some wordsmithing on this line. @swaminator

Reply:
Thanks for the quick turn around... id like to get @jordanranz ’s input on the changes i made to the typescript files though.

Reply:
1. Instead of a separate H2 title, I'd add it as a callout under ## Amplify Android
2. Change language: "Amplify for Android is the recommended way to start building a cloud connected app. If you are currently using the Mobile SDK for Android, access the documentation [here](~/sdk/sdk.md).